### DISCUSSION TITLE
Need to change the rotation of the model from horizontal to vertical axis in AR (WEBXR and Quicklook)

### DISCUSSION THREAD

**Discussion by elvispeter1991** [google/model-viewer#2293] [Q&A]
I need to change the rotation of the model from horizontal axis to vertical axis in Web XR Mode for android and Quick-look for iOS.

Can anyone help me out.

**Comment by elalish** [comment]
Models placed on the floor already rotate around their vertical axis. Can you describe what you mean a bit more? Perhaps with a screen recording?

**Reply by elvispeter1991** [reply]
Hi elalish,

I mean while placing the models on the wall in Web XR its unable to rotate it on either axis.

How can i enable rotation on this.

**Reply by elalish** [reply]
Ah, yes this is intended behavior. Can you describe your use case where you would like to be able to rotate something on a wall? It would not be hard to add, it just seemed unnecessary.

**Reply by elvispeter1991** [reply]
yes..am adding a panel to the wall and the user can rotate it on the wall surface so they can change the orientation of the panel and check whether it looks good 

https://user-images.githubusercontent.com/45065114/115267610-fedba900-a156-11eb-938d-794033cfbaf1.mp4

**Comment by ronpadz** [comment]
I have a use case for this as well. Cabinet hardware (handles) can often be mounted horizontally or vertically. I'd like to give the user the ability to see it both ways by rotating it on the vertical plane.

**Reply by elalish** [reply]
For that, I'd recommend a button that changes our `orientation` attribute: https://modelviewer.dev/docs/#entrydocs-scenegraph-attributes-orientation
